eng/liveBuilds.targets

```diff
@@ -118,6 +118,16 @@
       <MonoIncludeFiles Include="$(MonoArtifactsPath)\include\**\*.*" />
     </ItemGroup>
 
+    <!-- Host files. Mobile uses a different hosting model, so we don't include the .NET host components there. -->
+    <ItemGroup Condition="'$(TargetsMobile)' != 'true' and Exists('$(DotNetHostBinDir)')">
```

Comment:
@ViktorHofer This is the problematic part. By moving the host files to RuntimeFiles they ended up getting copied into the shared framework pack, when they were only supposed to be in the runtime pack. Keeping this in the Microsoft.NetCore.App package fixed the problem.

Reply:
Also, the changes in `externals.proj` basically replicate the behavior that was intended with the above change.